test/functional/interface_zmq.py

```diff
@@ -70,9 +72,9 @@ def receive_sequence(self):
         label = chr(body[32])
         mempool_sequence = None if len(body) != 32+1+8 else struct.unpack("<Q", body[32+1:])[0]
         if mempool_sequence is not None:
-            assert label == "A" or label == "R"
+            assert_true(label == "A" or label == "R", err_msg="{} different from A or C".format(label))
```

Comment:
Could optionally also implement `assert_in` (see https://docs.python.org/3/library/unittest.html#assert-methods)?

Reply:
f-strings are easier to read imo:
```suggestion
            assert_true(label == "A" or label == "R", err_msg=f"{label} different from A or C")
```

Reply:
ops, good catch, thanks!